src/crypto/mod.rs

```diff
@@ -71,3 +156,74 @@ pub trait SigningPrivateKey: SigningPublicKey {
     /// Given a digest, returns a signature
     fn sign(&self, digest: &[u8]) -> Result<Vec<u8>, CoseError>;
 }
+
+/// Values from https://tools.ietf.org/html/rfc8152#section-8.1
+#[derive(Debug, Copy, Clone, Serialize_repr, Deserialize_repr)]
+#[repr(i8)]
```

Comment:
Based on the [spec](https://datatracker.ietf.org/doc/html/rfc8152#section-16.4), I think this should be at least `i32`

Reply:
That piece of code was moved from [sign.rs:17-26](https://github.com/awslabs/aws-nitro-enclaves-cose/pull/36/files#diff-f54f613d47e227dca4e28dab3a680b05ec273a55d30c2669b90483ecca740d6cL17-L26). This isn't the right PR to fix it. Should I create a separate PR?